index.html

```diff
@@ -11,6 +11,17 @@
 
     <header class="app-header">
         <a href="#" class="logo">Narson</a>
+
+        
+        <div id="myNav" class="overlay">
+            <a href="javascript:void(0)" class="closebtn">&times;</a>
+            <div class="overlay-content">
+                <a id="portfolio" href="#work">Portfolio</a>
+                <a id="about" href="#about">About</a>
+                <a id="contact" href="#">Contact</a>
+            </div>
+        </div>
+
```

Comment:
Your design
![image](https://github.com/rbd3/Narson-Portfolio/assets/78251927/a7b6b32a-afac-4270-b3ca-f8c97c3f98af)

Figma
![image](https://github.com/rbd3/Narson-Portfolio/assets/78251927/6dad2365-7e10-4a2b-8589-4ffc065c3dfd)

According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md), it is important that we stick to the design as much as possible using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- [ ] Kindly modify the styling of the background by making it transparent and blurry so as to match the given template. 💪